src/OSPSuite.Core/Domain/Descriptors/EntityDescriptorMap.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace OSPSuite.Core.Domain.Descriptors
+{
+   public class EntityDescriptorMap<T> where T : IEntity
```

Comment:
Convenience method to have the entity AND the descriptor together. Entity could be saved in the descriptor but I felt this was easier to handle that way